autoload/phpactor/mapping.vim

```diff
@@ -0,0 +1,101 @@
+xnoremap <silent> <Plug>phpactorExtractExpression
+    \ :<C-U>call phpactor#ExtractExpression(visualmode())<CR>
+
+nnoremap <silent> <Plug>phpactorExtractConstant
+    \ :call phpactor#ExtractConstant()<CR>
+
+nnoremap <silent> <Plug>phpactorExtractMethod
+    \ :set opfunc=phpactor#ExtractMethod<CR>g@
+xnoremap <silent> <Plug>phpactorExtractMethod
+    \ :<C-U>call phpactor#ExtractMethod(visualmode())<CR>
+
+nnoremap <silent> <Plug>phpactorImportUseStatement
+    \ :call phpactor#UseAdd()<CR>
+
+nnoremap <silent> <Plug>phpactorGotoDefinition
+    \ :call phpactor#GotoDefinition()<CR>
+nnoremap <silent> <Plug>phpactorGotoDefinitionVsplit
+    \ :call phpactor#GotoDefinitionVsplit()<CR>
+nnoremap <silent> <Plug>phpactorGotoDefinitionHsplit
+    \ :call phpactor#GotoDefinitionHsplit()<CR>
+nnoremap <silent> <Plug>phpactorGotoDefinitionTab
+    \ :call phpactor#GotoDefinitionTab()<CR>
+
+nnoremap <silent> <Plug>phpactorContextMenu
+    \ :call phpactor#ContextMenu()<CR>
+
+nnoremap <silent> <Plug>phpactorNavigate
+    \ :call phpactor#Navigate()<CR>
+
+nnoremap <silent> <Plug>phpactorTransform
+    \ :call phpactor#Transform()<CR>
+
+nnoremap <silent> <Plug>phpactorClassExpand
+    \ :call phpactor#ClassExpand()<CR>
+
+nnoremap <silent> <Plug>phpactorHover
+    \ :call phpactor#Hover()<CR>
+
+
+" Function to be called to initializes the mappings for every buffer
+function! phpactor#mapping#init()
+    if v:false == get(g:, 'phpactorActivateMappings', v:false)
+      return
+    endif
+
+    " Mnemotechnic "Change into Variable"
+    call phpactor#mapping#map('cv', '<Plug>phpactorExtractExpression', 'n')
+
+    " Mnemotechnic "Change into Method"
+    call phpactor#mapping#map('cm', '<Plug>phpactorExtractMethod', 'n')
+
+    call phpactor#mapping#map('yu', '<Plug>phpactorImportUseStatement', 'n')
+
+    if v:true == get(g:, 'phpactorActivateOverlapingMappings', v:false)
+        " Visual mapping starting with `c` prevent to change a selection
+        " Use `s` instead to replace a selection while the mappings are activated
+        call phpactor#mapping#map('cv', '<Plug>phpactorExtractExpression', 'x')
+        call phpactor#mapping#map('cm', '<Plug>phpactorExtractMethod', 'x')
+
+        " `cc` replace the current line, use `S` instead
+        " Mnemotechnic "Change into Constant"
+        call phpactor#mapping#map('cc', '<Plug>phpactorExtractConstant', 'n')
+
+        " Replaces the tags mappings
+        call phpactor#mapping#map('<C-]>', '<Plug>phpactorGotoDefinition', 'n')
+        call phpactor#mapping#map('<C-w>]', '<Plug>phpactorGotoDefinitionVsplit', 'n')
+        call phpactor#mapping#map('<C-w><C-]>', '<Plug>phpactorGotoDefinitionVsplit', 'n')
+    endif
+
+    " Put in last to override any default mapping if needed
+    for mapping in get(g:, 'phpactorCustomMappings', [])
+        call call('phpactor#mapping#map', add(copy(mapping), v:true))
+    endfor
+endfunction
+
+" vim: et ts=4 sw=4 fdm=marker
```

Comment:
I guess this is a leak from the other PR?

Reply:
I think so, I had remove my own mappings for the mapping branch in prevision of a PR.
So I rebased this PR on top of it to still some mappings during dev.
I just rebased the branch on top of the current develop and pushed it back.